app/lib/backend_api_logic/metric_extension.rb

```diff
@@ -10,9 +10,8 @@ module MetricExtension
       validate :unique_extended_system_name, if: :backend_api_metric?
       before_save :extend_system_name, if: :backend_api_metric?
 
-      def system_name
-        attributes['system_name'].to_s.gsub(/#{Regexp.escape(SYSTEM_NAME_SUFFIX_SEPARATOR)}\d+\z/, '')
-      end
+      alias_method :system_name, :system_name_without_suffix
+      public :system_name
```

Comment:
Do we need this line? :scream: why?

Reply:
Because `#system_name_without_suffix` is protected.